.buildkite/scripts/steps/scalability/benchmarking.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+#.buildkite/scripts/bootstrap.sh
+echo "--- yarn kbn reset && yarn kbn bootstrap"
+yarn kbn reset && yarn kbn bootstrap
+
```

Comment:
I'm getting error on running `.buildkite/scripts/bootstrap.sh`, maybe worker should be configured to use bazel cache?

Reply:
I tried and it still fails:

```


yarn run v1.22.19
--
  |   | $ node scripts/kbn bootstrap --force-install
  | 2022-09-01 12:17:52 UTC | HINT
  | 2022-09-01 12:17:52 UTC | HINT If experiencing problems with node_modules try `yarn kbn bootstrap --force-install` or as last resort `yarn kbn reset && yarn kbn bootstrap`
  | 2022-09-01 12:17:52 UTC | HINT
  | 2022-09-01 12:17:52 UTC | ERROR  [bootstrap] failed: [bazel] exited with code [2]
  | 2022-09-01 12:17:52 UTC | output:
  | 2022-09-01 12:17:52 UTC | INFO: Invocation ID: 0c5df2b5-6e60-4b1c-bce4-add0ba5d0757
  | 2022-09-01 12:17:52 UTC | ERROR: Failed to init auth credentials: The Application Default Credentials are not available. They are available if running in Google Compute Engine. Otherwise, the environment variable GOOGLE_APPLICATION_CREDENTIALS must be defined pointing to a file defining the credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
  | 2022-09-01 12:17:52 UTC | ERROR: Error initializing RemoteModule}
  |   | error Command failed with exit code 1.
  |   | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
  | 2022-09-01 12:17:52 UTC | 🚨 Error: The command exited with status 1
```